core/common/assets/js/api/core/shortcuts.js

```diff
@@ -120,10 +130,19 @@ export default class Shortcuts {
 		}
 
 		const namespace = component.getNamespace(),
-			rootScope = component.getRootContainer();
+			namespaceRoot = component.getRootContainer();
+
+		const filteredByNamespace = scopes.some( ( scope ) => {
+			return namespace === scope;
+		} );
```

Comment:
You can shorten it: `const filteredByNamespace = scopes.some( ( scope ) => namespace === scope );`